projects.go

```diff
@@ -646,6 +646,24 @@ func (s *ProjectsService) ShareProjectWithGroup(pid interface{}, opt *ShareWithG
 	return s.client.Do(req, nil)
 }
 
+// UnshareProjectFromGroup allows to unshare a project from a group.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/projects.html#delete-a-shared-project-link-within-a-group
+func (s *ProjectsService) UnshareProjectFromGroup(pid interface{}, groupID int, options ...OptionFunc) (*Response, error) {
```

Comment:
I think a name like `DeleteSharedProjectFromGroup` is a much closer match to description in the API docs. And to make methods easy searchable, I prefer to keep as close to the docs as possible.